app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
```

Comment:
- [ ] Great job  enabled `:confirmable` is in the user model 👏🏻 However, it doesn't work as expected, the app keeps asking me to confirm my email, although I've already done so more than twice using the send link in the terminal. please solve this problem. 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/234951414-8c7d1151-eb0c-4ea3-a298-79c0e88b71ad.png)


- [ ] The same thing here 👇🏻 I'm geeting the reset password link but I can't change it.  Please make sure A user can reset the password in the application. 💯 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/234953189-5da164a0-3828-4bbb-b3f1-8f7599eb0b6f.png)